### PR TITLE
Jellis -> Master (Draft of client Demo Presenations)

### PR DESCRIPTION
**Pull request by palamedes**
Rough Draft of Client Demo Presentations
Linked it
and made suggestions / observations about the plan.

### CODE REVIEW

**Comment by joshfreemanIO** [comment]
Don't allow ambiguity to sneak in–

>​˜Of course, it goes without saying that we should˜​ strive to present to the client a working version of their product being used exactly how they would use it and show it working in the ma
>
> ~The "dry run" of the presentation ~should~ be performed exactly as if the client were on the call and all aspects of the presentation should be done.~
> Perform the _dry run_ of the presentation using the same flow and script that will be presented to the client

**Comment by joshfreemanIO** [comment]
I think we should remove the "room mic/jambox" bit–specifying the studio is sufficient, and won't make any sense for new people who never experienced it

**Comment by palamedes** [comment]
Fixed first ambiguity stuff.. but I want to keep the mic thing in for now since those who are here have experienced it but dont realize how bad it sounds when you're not in the room. I want to drive this point home.. it sounds BAD.

**Comment by joshfreemanIO** [comment]
I love the idea of calling them presentations–that's what they really are (and always have been) but it helps convey what they are

**Comment by palamedes** [comment]
Yep, this document adds a lot of structure, I realize.. but it's for the over all good.

File: PLAN.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ The primary objective is to codify **_The Grok Way_** of software development an
 
 The secondary objective is to serve as a recruitment and advertising tool that provides transparency and insight into the organization. The ability to clearly communicate _how_ Grok conducts operations will force introspection and will demonstrate the seriousness and professionalism we bring to software development.
 
+### Instructions on how to update the grok way
+
+Each individual topic should exist in it's own markdown file within `the-grok-way` repository.
+
+
+{ Jacob here is where you should further define all things.. Each a file?  Each a directory? All caps file structure?  Whats the naming scheme?  How are things organized?  We need an index or way to find things?  etc.. }
```
Comment (Jacob-Ernst) [inline PLAN.md]: @palamedes I was thinking maybe use all caps and words separated by hyphens for filenames from now on. For example `EXAMPLE-FILE.md`. 

I also think we should split these topics into single category files. Meaning that "SASS/CSS Styleguide" would be organized inside one markdown file, but "Continuing Education" and "Social Media" would be organized inside separate files. Thoughts?

Comment (palamedes) [inline PLAN.md]: The only issue there is if we want to add files, like Lauren is doing, we would need some directory structure to handle that.

Comment (Jacob-Ernst) [inline PLAN.md]: Well ultimately this is only helpful until we have an actual design direction from Josh Mathis to base directory structure on. The only intent behind these initial files is for communication of thoughts and ideas on topics.

**Pull request merged**